doc/ref_cert/RC2/conf.py

```diff
@@ -0,0 +1,13 @@
+project = 'Anuket Reference Conformance for Kubernetes (RC2)'
+copyright = '2021, Anuket'
```

Comment:
Anuket contributors

Reply:
It's usually the foundation or the first company contributing to it. Is that the reason of -1?!
I